tests/unit/models/test_utils.py

```diff
@@ -11,3 +32,24 @@ class TestUtils(TestCase):
     def test_kwargs_req(self):
         with self.assertRaises(XRPLModelException):
             IssuedCurrency(currency, issuer)
+
+    def test_throws_if_positional_args_mixed_with_non_positional_args(self):
+        with self.assertRaises(XRPLModelException):
```

Comment:
I'm confused why this works with >=3.10 - shouldn't it be throwing a different error?

Reply:
The old decorator should still be used in some form, to keep support for the error for <3.10.